eng/pipelines/maintenance-packages.yml

```diff
@@ -96,7 +96,46 @@ stages:
               $(_InternalBuildArgs)
             displayName: $(_BuildJobDisplayName)
 
+- stage: pack
+  displayName: Pack
```

Comment:
I see what this did: It created a completely separate section after Build. Do we want this or should I include the pack work in the Build stage?

![image](https://github.com/dotnet/maintenance-packages/assets/1175054/8a37a706-c55c-4cb6-970f-bf3a600bab3a)

Reply:
Maybe we do want to put it inside Build. Otherwise I don't see it showing up in the main PR screen:

<img src="https://github.com/dotnet/maintenance-packages/assets/1175054/6ce11f89-b7fc-482a-8c13-3b497cfeb78a" width="600px" />

Reply:
You should see the stage in GH but it currently might only run after the default Build stage. You might need to set some stage parameter here to indicate that it should run in parallel with the default Build stage.

Reply:
I found that setting an empty array of dependsOn should make it run in parallel